<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pdu_SetSwitchDlg</class>
 <widget class="QDialog" name="Pdu_SetSwitchDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>302</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 11pt &quot;微软雅黑&quot;;
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>开关选择</string>
   </property>
   <widget class="QRadioButton" name="openRadio">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>开</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="closeRadio">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>关</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="uniteCheckBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>全开/全关</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="wholeCheckBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>全局生效</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>120</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>461</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>输出位开关设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cancelBtn</tabstop>
  <tabstop>openRadio</tabstop>
  <tabstop>closeRadio</tabstop>
  <tabstop>uniteCheckBox</tabstop>
  <tabstop>wholeCheckBox</tabstop>
  <tabstop>saveBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>Pdu_SetSwitchDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
